upa/src/sdk/submissionIntervals.ts

```diff
@@ -131,7 +134,6 @@ export function mergeSubmissionIntervals<T>(
 
   const mergedIntervals: SubmissionInterval<T>[] = [];
   let curInterval = intervals[0];
-  let doNotMerge = false;
 
   for (let i = 1; i < intervals.length; ++i) {
```

Comment:
Line 142 comments maybe update to include info about submissions with multiple dummys.